lib/rdoc/generator/template/rails/search_index.rhtml

```diff
@@ -1,5 +1,8 @@
-<html>
-    <head>File index</head>
```

Comment:
I wasn't 100% why this was `<head>File index</head>`, but it looked incorrect to me. So I wrapped the text in a `<title>` tag.

Reply:
Yes, that seems wrong :)